__tests__/app.test.js

```diff
@@ -131,4 +131,67 @@ describe("CORE: GET /api/articles error test suite", () => {
             expect(body.msg).toBe("Path not found.");
         })
     });
+});
+describe("CORE: POST /api/articles/:article_id/comments functionality test suite", () => {
+    test("returns a 201 POST with a comment object of the posted comment to the given article", () => {
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send({ "username": "lurker", "body": "BANANA!" })
+        .expect(201)
+        .then(({ body }) => {
+            expect(body.comment).toHaveProperty("author");
+            expect(body.comment).toHaveProperty("body");
+            expect(body.comment).toHaveProperty("article_id");
+            expect(body.comment).toHaveProperty("comment_id");
+            expect(body.comment).toHaveProperty("created_at");
+            expect(body.comment).toHaveProperty("votes");
+        })
+    })
```

Comment:
do one more happy path - ignored unnecessary properties on the sent comment (i.e. age:100)